aspnetcore/mvc/razor-pages/razor-pages-convention-features/sample/Pages/About.cshtml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ModelProvidersSample.Pages
+{
+    public class AboutModel : PageModel
+    {
+        public string Message { get; private set; }
+
+        public string RouteDataGlobalAttribute { get; private set; }
+
+        public string RouteDataAboutAttribute { get; private set; }
+
+        public void Get()
+        {
+            Message = $"Your application description page.";
+
+            #region snippet1
+            if (RouteData.Values["globalAttribute"] != null)
+            {
+                RouteDataGlobalAttribute = $"Route data for 'globalAttribute' was provided: {WebUtility.HtmlEncode(RouteData.Values["globalAttribute"].ToString())}";
```

Comment:
Shouldn't need the `HtmlEncode`. Razor would do this for you (currently this gets double encoded).

Reply:
A couple of places with the same issue.

Reply:
Ok ... thanks. Just cover'in my b-hind here. :smile: I forgot that detail of working with route data ... it's been a loooong while since I've had to think about piping route data values back into the UI. I'll remove these throughout the sample.